database/migrations/2017_12_04_113032_create_families_table.php

```diff
@@ -16,7 +16,6 @@ public function up()
         Schema::create('families', function (Blueprint $table) {
             $table->increments('id');
             $table->string('rvid')->nullable(); // Rose Voucher ID; globally unique per participant.
-            $table->string('surname');
```

Comment:
You have not released any yet, right? OK to edit migrations if that's the case.

Reply:
Ther eis a staging box, which is "release on merge" but it's hideously broken anyway, and we'll formally tidy it up when i;ve got all the pages templated.